src/Repository/DistrictRepository.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use AppBundle\Entity\District;
+use AppBundle\Entity\ReferentTag;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\ORM\Query\Expr\Join;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+class DistrictRepository extends ServiceEntityRepository
+{
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, District::class);
+    }
+
+    /**
+     * Finds referent tag for district by coordinates of the point.
+     *
+     * @return ReferentTag
+     */
+    public function findReferentTagForDistrictByCoordinates($latitude, $longitude): ?ReferentTag
+    {
+        return $this->getEntityManager()->createQueryBuilder()
+            ->from(ReferentTag::class, 'tag')
+            ->select('tag')
+            ->innerJoin(District::class, 'district', Join::WITH, 'district.referentTag = tag')
+            ->innerJoin('district.geoData', 'geoData')
+            ->where("ST_Within(ST_GeomFromText(CONCAT('POINT(',:latitude,' ',:longitude,')')), geoData.geoShape) = 1")
```

Comment:
can you try to use Doctrine Point type?
```php
->where('ST_Within(:point, geoData.geoShape) = 1')
->setParameter('point', new Point($latitude, $longitude))
```

Reply:
It doesn't work and throws an error `Geometry byte string must be little endian`.